gcc/rust/typecheck/rust-hir-type-check-expr.cc

```diff
@@ -1017,6 +1017,19 @@ TypeCheckExpr::visit (HIR::StructExprStruct &struct_expr)
       return;
     }
 
+  TyTy::ADTType *adt = static_cast<TyTy::ADTType *> (struct_path_ty);
+  for (auto variant : adt->get_variants ())
+    {
+      if (!variant->get_fields ().empty ())
+	{
+	  rust_error_at (struct_expr.get_locus (), ErrorCode::E0063,
+			 "missing field %s in initializer of %<%s%>",
```

Comment:
```suggestion
			 "missing field %s in initializer of %qs",
```
this is a shorter version of the same thing